src/EditorFeatures/CSharpTest/CodeActions/InitializeParameter/AddParameterCheckTests.cs

```diff
@@ -2740,7 +2740,7 @@ public C(string s)
                 { CSharpCodeStyleOptions.AllowEmbeddedStatementsOnSameLine, false },
             },
             CodeActionIndex = 1,
-            CodeActionEquivalenceKey = nameof(FeaturesResources.Add_string_IsNullOrEmpty_check)
+            CodeActionEquivalenceKey = $"{nameof(FeaturesResources.Add_0_check)};string.IsNullOrEmpty"
```

Comment:
honestly, just generate teh same CAEK as before.  it shoul dbe easy to do.  it's just "Add_string_IsNullOrEmpty_check"